tools/test/audio/test_utils/test_run.m

```diff
@@ -92,14 +92,17 @@
 	fprintf(fh, 'XTRUN=\n');
 end
 
-% Override defaults in comp_run.sh
-fprintf(fh, 'VALGRIND=false\n');
 fclose(fh);
 
 arg = sprintf('-t %s', fn_config);
 rcmd = sprintf('%s %s', ex, arg);
 fprintf('Running ''%s''...\n', rcmd);
-system(rcmd);
+ret = system(rcmd);
+if ret
+	fprintf('Returned status %d\n', ret);
```

Comment:
Because rcmd may print stuff too:

```suggestion
printf('%s eturned status %d\n', rcmd, ret);
```
Can't you combine `fprintf` and `error` into one? I hope one does not print on stdout while the other on stderr because buffering could order them randomly.

Reply:
Re-read the manual, today error in octave can do same as printf... and so does Matlab. It was an old limitation that no more bothers. Good that you asked!